src/simsopt/field/normal_field.py

```diff
@@ -47,44 +53,52 @@ def __init__(self, nfp=1, stellsym=True, mpol=1, ntor=0,
         if vns is None:
             vns = np.zeros((self.mpol + 1, 2 * self.ntor + 1))
 
-        if vnc is None and not stellsym:
+        if not self.stellsym and vnc is None:
             vnc = np.zeros((self.mpol + 1, 2 * self.ntor + 1))
+        
+        if computational_boundary is None:
+            from simsopt.geo import SurfaceRZFourier
+            computational_boundary = SurfaceRZFourier(nfp=nfp, stellsym=stellsym, mpol=mpol, ntor=ntor)
+        computational_boundary.fix_all()
```

Comment:
This seems like potentially dangerous and undocumented behavior. There might be use cases for NormalField unrelated to spec (maybe QFM surfaces?) where one would want the surface shape to change.